requre/online_replacing.py

```diff
@@ -472,3 +472,22 @@ def recording_requests(
         storage_file=storage_file,
     ) as cassette:
         yield cassette
+
+
+def apply_decorator_to_class_methods(decorator, regexp_method_pattern="test.*"):
```

Comment:
Do we want to have `"test.*"` as a default? I would use `.*` or `None` (=skip check) by default since the name does not say anything about tests. (And/or we can have one for tests with that `test` prefix.)

Reply:
And  `class_method` is here a bit misleading term since it can mean both methods of the class and class-methods.

```suggestion
def apply_decorator_to_all_methods(decorator, regexp_method_pattern="test.*"):
```

Reply:
> Do we want to have `"test.*"` as a default? I would use `.*` or `None` (=skip check) by default since the name does not say anything about tests. (And/or we can have one for tests with that `test` prefix.)

I suggest to have it there because I expect the main usage in with testing, and this helps us to avoid more multiple storing via this via some helper functions, e.g.  you will have some `def check_it(self)` and there will be `test1` and `test2` what will use this `check_it` it means that in case of .* it decorates also check and values will be doubled in files, similar to my test, what does `self.test1()` as part of `test2` and output file for test2 contains stored values via test1 decorators and test2 :-) it is little bit unhappy, but probably not anyhow possible to avoid it fully, but at least this default filter helps us to avoid decorate all methond what can contain just some shared code. and not test itself

Reply:
My problem is with the name that it does not match the behaviour...